awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -190,6 +190,15 @@ def load_config(self, config_path):
                 else:
                     setattr(self, honorred_setting, config_data[honorred_setting])
 
+    @staticmethod
+    def param_to_endpoint(name):
+        exceptions = {
+            'inventory': 'inventories',
+            'target_team': 'teams',
+            'workflow': 'workflow_job_templates'
+        }
+        return exceptions.get(name, '{0}s'.format(name))
```

Comment:
@john-westcott-iv, this is my suggested alternative to

https://github.com/ansible/awx/pull/6353/files#diff-72d907817eec8312fdbe88c93d20b610

I have no reservations about adding `"webhook_credentials": "credentials"`. In that event, it would still be doing exactly what it claims to do. It is receiving a `tower_*` module parameter, and giving a global endpoint corresponding to its type. If a param name is reused for multiple types, then that would break this, but no one should do that.